modules/sync/syncs/httpsynchronization.h

```diff
@@ -110,6 +117,16 @@ class HttpSynchronization : public ResourceSynchronization {
 
     // The thread that will be doing the synchronization
     std::thread _syncThread;
+
+    //The files that have already been synchronized
+    std::vector<std::string> _existingSyncedFiles;
+
+    //The files that have been synchronized this time
+    std::vector<std::string> _newSyncedFiles;
+
+    const std::string _ossyncVersionNumber = "1.0";
+
+    const std::string _synchronizationToken = "Synchronized";
```

Comment:
```suggestion
    constexpr std::string_view _synchronizationToken = "Synchronized";
```

Also for both of these, they'd be defined at the top of the `private` block before the function parameters.  Even better would be to move their definitions into the anonymous namespace inside the `cpp` since noone who imports this header really needs to care about the actual values;  so the cpp would have:

```cpp
namespace {
    constexpr std::string_view OSSyncVersionNumber = "1.0";
    constexpr std::string_view SynchronizationToken = "Synchronized";
} // namespace
```

at the top.  Usually the less visibility, the better